aperturedb/Utils.py

```diff
@@ -617,45 +617,30 @@ def remove_all_objects(self):
 
         cmd = {"constraints": {"_uniqueid": ["!=", "0.0.0"]}}
 
-        queries = [
-            [{"DeleteEntity": cmd}],
-            [{"DeleteImage": cmd}],
-            [{"DeleteVideo": cmd}],
-            [{"DeleteBlob": cmd}],
+        transaction = [
+            {"DeleteEntity": cmd},
```

Comment:
DeleteEntity should be the last thing. Otherwise, based on the constraints on `cmd`, DeleteEntity will remove all the nodes, including _Image, _Videos, Etc, leaving them file in the file system. 

We need to do DeleteImage, DeleteVideo, etc first, and leave DeleteEntity for last. 

This is a good time to make that fix?

Reply:
OK will do.